upgrade/v102to110/upgrade.go

```diff
@@ -0,0 +1,129 @@
+package v102to110
+
+import (
+	"fmt"
+	"github.com/longhorn/longhorn-manager/util"
+	"github.com/pkg/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+
+	"github.com/longhorn/longhorn-manager/types"
+
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta1"
+	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+)
+
+// This upgrade is needed because we use separate the disk resources from node
+// object (in v1.0.2) and use CRs to track them (in v1.1.0).
+// Since the naming of the disk resources may change, the related replicas
+// should be updated, too.
+// Link to the original issue: https://github.com/longhorn/longhorn/issues/1269
+
+const (
+	upgradeLogPrefix = "upgrade from v1.0.2 to v1.1.0: "
+)
+
+func UpgradeCRDs(namespace string, lhClient *lhclientset.Clientset) error {
+	if err := createDisksAndUpdateNodes(namespace, lhClient); err != nil {
+		return err
+	}
+	if err := doReplicaUpgrade(namespace, lhClient); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func createDisksAndUpdateNodes(namespace string, lhClient *lhclientset.Clientset) (err error) {
+	defer func() {
+		err = errors.Wrapf(err, upgradeLogPrefix+"upgrade instance manager failed")
+	}()
```

Comment:
Is the log message correct? I got into this log message while upgrading and it was a bit confusing.